main.rb

```diff
@@ -1,22 +1,52 @@
 require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
-require_relative 'classroom'
 require_relative 'student'
 require_relative 'rental'
 require_relative 'book'
+require_relative 'app'
+def main
+  choice_list
+  nbr = gets.chomp.to_i
+  list_cases(nbr)
+  main
+end
+def choice_list
+  puts 'Please choose an option by entering a number'
+  puts '1 - List all Books'
+  puts '2 - List all Person'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a Rental'
+  puts '6 - List all Rentals for a given person'
+  puts '7 - Exit'
+end
+def list_cases(choice)
+  options = {
+    1 => :list_book,
+    2 => :list_person,
+    3 => :create_person,
+    4 => :create_book,
+    5 => :create_rental,
+    6 => :list_rental,
+  }
```

Comment:
- [ ] At the moment, users cannot safely exit the app without an error as shown below 👇 Please handle the option of a user exiting the app by inputing option '7'
![image](https://github.com/touredri/library/assets/46839548/e2c4d4cf-8036-44a6-af13-f880bbd9e860)

Reply:
Ohh I forget it